### DISCUSSION TITLE
Add google translate or let it to work. Thanks.

### DISCUSSION THREAD

**Discussion by Valdesca** [hodcroftlab/covariants#292] [Ideas]
https://translate.google.es/?hl=es&sl=auto&tl=es&op=websites 
On this web to translate, pasting link from covariants result on blank web.
like this:
https://covariants-org.translate.goog/variants/20A.EU2?_x_tr_sl=auto&_x_tr_tl=es&_x_tr_hl=es&_x_tr_pto=wapp 
Thanks for the great work on this variant's web.

**Comment by emmahodcroft** [comment]
Hm, this seems to be related to something about
```
'https://covariants-org.translate.goog' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

Does that mean anything to you @ivan-aksamentov ? I hadn't realised CoVariants.org didn't work on Google Translate.

**Reply by ivan-aksamentov** [reply]
@emmahodcroft No. I never knew that existed. I think translate.google.com only works with very simple HTML websites. We have a bunch of dynamic JavaScript stuff.

**Comment by ivan-aksamentov** [comment]
@Valdesca You could try translation feature in the Chrome browser context menu:

![01](https://user-images.githubusercontent.com/9403403/158916123-3e012c06-92d8-4470-b2a0-1c3db8ca905c.png)

**Comment by emmahodcroft** [comment]
@Valdesca I'm afraid I'm not sure this is something we can support right now, as I don't think we have the expertise to tackle this. However, thanks for letting me know - I didn't realise it didn't work! And it would be nice to support. Let's leave the discussion open in case someone happens to come by and can suggest an easy fix (perhaps there is a setting we could change).

In the meantime, as Ivan said, I'd suggest trying Google Chrome, which is very good at doing translations of websites!
